Tests/SentryTests/Helper/SentrySwizzleTests.m

```diff
@@ -353,4 +357,23 @@ - (void)testSwizzleDontCallOriginalImplementation
     XCTAssertThrows([a methodForSwizzlingWithoutCallOriginal]);
 }
 
+- (void)testUnswizzleInstanceMethod
+{
+    SEL methodForUnswizzling = NSSelectorFromString(@"methodForUnswizzling");
+
+    SentrySwizzleTestClass_A *object = [SentrySwizzleTestClass_B new];
+    swizzleVoidMethod(
+        [SentrySwizzleTestClass_A class], methodForUnswizzling, ^{ SentryTestsLog(@"A"); },
+        SentrySwizzleModeAlways, (void *)methodForUnswizzling);
+    [object methodForUnswizzling];
+    ASSERT_LOG_IS(@"A");
+
+    [SentryTestsLog clear];
+
+    [SentrySwizzle unswizzleInstanceMethod:methodForUnswizzling
```

Comment:
`m`: I would like to have some test cases for calling unswizzleInstanceMethod for non swizzled methods and calling unswizzleInstanceMethod twice.